src/foam/i18n/ClientCacheTranslationService.js

```diff
@@ -84,6 +91,10 @@ foam.CLASS({
         // source: l.source,
         target: l.target
       }));
+      // populate map entries
```

Comment:
Remove the code above. Remove the check, otherwise you won't override language global messages with variant specific ones.